zp-relayer/services/loggerMiddleware.ts

```diff
@@ -1,9 +1,31 @@
-import winston from 'winston'
+import { format, transports } from 'winston'
 import expressWinston from 'express-winston'
+import config from '@/config'
+import { logger } from './appLogger'
 
-export function createLoggerMiddleware(filename: string = 'zp.log') {
+export function createPersistentLoggerMiddleware(filename: string = 'zp.log') {
   return expressWinston.logger({
-    transports: [new winston.transports.File({ filename })],
-    format: winston.format.combine(winston.format.json()),
+    transports: [new transports.File({ filename })],
+    format: format.combine(format.json()),
+  })
+}
+
+export function createConsoleLoggerMiddleware() {
+  return expressWinston.logger({
+    winstonInstance: logger,
+    level: 'debug',
+    ignoredRoutes: config.logIgnoreRoutes,
+    headerBlacklist: [
+      'accept',
```

Comment:
Does it make sense to provide the ability to configure the list of the headers through a environment variable?

Reply:
Yeah, probably makes sense, leaving current header blacklist entries as default ones.